src/app/public/styles/_forms.scss

```diff
@@ -8,12 +8,14 @@
   font-size: $sky-font-size-base;
   color: $sky-text-color-default;
 
-  /* Remove device-specific styles */
-  -webkit-appearance: none;
-  -moz-appearance: none;
-  appearance: none;
-  border-radius: 0;
-  -webkit-border-radius: 0;
+  &:not([type="range"]):not([type="radio"]):not([type="checkbox"]){
```

Comment:
After looking at how many input types there are, I'm thinking that the non-text-based inputs should not be affected by the `.sky-form-control` class. We should probably share this screenshot with @Blackbaud-ToddRoberts  to see what his opinion is; I would assume that only the text-based inputs should be styled with `.sky-form-control`, and the other ones left alone.

![screen shot 2018-10-24 at 5 36 43 pm](https://user-images.githubusercontent.com/12497062/47463139-65b0bf00-d7b3-11e8-9e43-97b9160022b6.png)

Reply:
For example, inputs with type `button`, `checkbox`, `color`, `file`, `radio`, `range`, `reset`, `submit` should be unaffected by `.sky-form-control`.

Reply:
Just by way of documenting our discussion, let's pull out this single line of code into a patch release, and worry about the visual tests later on.